lms/djangoapps/courseware/views/views.py

```diff
@@ -849,7 +836,7 @@ def course_about(request, course_id):
         # - Student is already registered for course
         # - Course is already full
         # - Student cannot enroll in course
-        active_reg_button = not(regularly_registered or is_course_full or not can_enroll)
+        active_reg_button = not ((registered and request.user.is_registered())or is_course_full or not can_enroll)
```

Comment:
seems like you have some weird whitespacing here?

Fixed below:
```python
active_reg_button = not((registered and request.user.is_registered()) or is_course_full or not can_enroll)
```

Reply:
I don't think so.

`not` is a keyword, not a function call.

Reply:
there's definitely a space missing after the `((registered and request.user.is_registered())`

Reply:
Fixed.